src/main/java/seedu/address/ui/MainWindow.java

```diff
@@ -33,6 +36,8 @@
 
     private static final String ICON = "/images/lisalogo.png";
     private static final String FXML = "MainWindow.fxml";
+    private static final String COMMANDBOX_TEXTFIELD_ID = "#commandTextField";
+    private static final String SEARCHBOX_TEXTFIELD_ID = "#searchTextField";
```

Comment:
dont we usually do this in the fxml file instead?

Reply:
These are identifiers which match the ones in fxml file. I need to use them to use lookup method to locate the two textfield directly from MainWindow